packages/api/tests/lambdas/test-sf-scheduler.js

```diff
@@ -86,7 +84,7 @@ test.serial('getProvider returns provider when input is a valid provider ID', as
 });
 
 test.serial('getCollection returns undefined when input is falsey', async (t) => {
```

Comment:
Update the test names through from `getCollection` -> `getApiCollection`

Reply:
Good call.  3b1b2759f